install/install-orange.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+set -ex
+
+
+OR_EXEC=`which openresty 2>&1`
+echo $OR_EXEC
+CHECK_OR_EXIST=`echo $OR_EXEC | grep ": no openresty" | wc -l`
+
+
+LUA_JIT_DIR=`$OR_EXEC -V 2>&1 | grep prefix | grep -Eo 'prefix=(.*?)/nginx' | grep -Eo '/.*/'`
+LUA_JIT_DIR="${LUA_JIT_DIR}luajit"
+echo $LUA_JIT_DIR
+
+
+LUA_ROCKS_EXEC=`which luarocks 2>&1`
+echo $LUA_ROCKS_EXEC
+CHECK_ROCKS_EXIST=`echo $LUA_ROCKS_EXEC | grep ": no luarocks" | wc -l`
+
+
+LUA_ROCKS_VER=`luarocks --version | grep -Eo  "luarocks [0-9]+"`
+echo $LUA_ROCKS_VER
+
+
+UNAME=`uname`
+echo $UNAME
+
+
+do_install() {
+    if [ "$UNAME" == "Darwin" ]; then
```

Comment:
do we need to support macOS?

Reply:
current not supported, I am correcting it.